connectors/v2/genie.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#music-player';
+Connector.artistSelector = '#ArtistNameArea';
+Connector.trackSelector = '#SongTitleArea';
+Connector.albumSelector = '#AlbumTilteArea';
+Connector.currentTimeSelector = '#playTime span';
+Connector.durationSelector = '#playTime strong';
+
+Connector.isPlaying = function() {
+	var btn = $('#PlayBtnArea')[0];
```

Comment:
@alexesprit I hadn't visited the site - but I was more commenting on the use of jquery to simplify the function.